### DISCUSSION TITLE
Add paid subscription!

### DISCUSSION THREAD

**Discussion by melnichek** [organicmaps/organicmaps#610] [Ideas]
I'd love to support the continued development of the app. Please add an option to have a paid subscription, even if it's just a donation with no features unlocked. I'm sure many users would love to pay for the app too.

**Comment by stephan75** [comment]
What about the sponsor feature on github.com?

**Comment by mds08011** [comment]
There are now multiple options to donate: https://organicmaps.app/donate/
- [Github Sponsorship](https://github.com/sponsors/organicmaps?o=esb)
- [Liberapay](https://liberapay.com/OrganicMaps/donate)
- [Paypal](https://www.paypal.com/donate?token=LKwNb2XdOsP2bU-N6MqJCUjf6AAVOAI6zyuJco4axth4MWg8NLvsrY_58fEay5Lb4VPOpqeQuegw4mfP)

**Reply by melnichek** [reply]
Thank you for the links. Works for me.

However I still think that it's in the best interest of the project to provide options to donate for less sophisticated users. The easiest is through in-app subscription.
